javascript/javascript1/week1/dog-age.js

```diff
@@ -0,0 +1,20 @@
+const dogYearOfBirth = 2017;
+let dogYearFutures = 2027;
+const humanYear = dogYearFutures-dogYearOfBirth;
+const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+let shouldShowResultInDogYear = true;
+
+if (shouldShowResultInDogYear) {
+console.log(`your dog will be ${dogYear} dog years old in ${dogYearFuture}`)
+}
+else {
+console.log(`your dog will be ${humanAge} years old in ${dogYearFuture}`)
+}
+
+shouldShowResultInDogYear = false;
+if (shouldShowResultInDogYear) {
+  console.log(`your dog will be ${dogYear} dog years old in ${dogYearFuture}`); 
+}
+else {
+  console.log(`your dog will be ${humanAge} dog years old in ${dogYearFuture}`);
+}
```

Comment:
This is really well organised! Functions will help you get rid of this repeated control flow :)